Ecommerce/ecommerce_app/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+
+class Product(models.Model):
+    product_name = models.CharField(max_length = 255)
```

Comment:
```suggestion
    product_name = models.CharField(max_length=255)
```

read about pep8 and django formatting, there is a package named black that handles these formatting changes